src/components/Sum/Sum.jsx

```diff
@@ -1 +1,13 @@
-// export const Sum = () => ();
+/* eslint-disable react/jsx-one-expression-per-line */
```

Comment:
It is generally a bad practice to disable ESLint rules unless absolutely necessary. You should consider re-enabling the react/jsx-one-expression-per-line rule and adjust your code to comply with it.